components/camel-aws/camel-aws-secrets-manager/src/main/docs/aws-secrets-manager-component.adoc

```diff
@@ -270,6 +270,66 @@ where `camel.vault.aws.refreshEnabled` will enable the automatic context reload,
 
 Note that `camel.vault.aws.secrets` is not mandatory: if not specified the task responsible for checking updates events will take into accounts or the properties with an `aws:` prefix.
 
+=== Automatic Camel context reloading on Secret Refresh with Eventbridge and AWS SQS Services
+
+There is also the possibility of using AWS Eventbridge in combination with AWS SQS Service.
```

Comment:
`Another option is to use AWS EventBridge in conjunction with the AWS SQS service.`